liferay-tango/liferay-portal
# LPS-145339 Replace useOnClickOutside hook with closeOnClickOutside property

Motivation
=======
In the Content Dashboard, if we have a categories popover opened and we click in another category label that opens a new popover, we want the first popover to hide.
[Link to ticket](https://issues.liferay.com/browse/LPS-145339)

Proposed Solution
========
We removed the useOnClickOutside hook and used closeOnClickOutside property instead. This way we remove unnecessary code and also fix the problem of closing one popover before opening a new one.

Steps to Verify*:
----------------

1. Add a vocabulary with at least one category
2. Add two basic web contents and assign them the categories created in step 1
3. Go to content dashboard and in the configuration select the vocabulary created in step 1
4. Check the table with all the contents
